components/datetime/src/format/time_zone.rs

```diff
@@ -53,6 +121,8 @@ where
 {
     let pattern = &time_zone_format
         .patterns
+        .as_ref()
+        .unwrap()
         .get()
```

Comment:
> suggestion 

Same here, we should use `expect()` instead of `unwrap()`

```rust
.expect("TimeZoneFormat::write_pattern() expected Some(patterns) but got None")
```